pkg/accountmanager/manager.go

```diff
@@ -257,3 +294,152 @@ func (a *AccountManager) getAccountPoller(name *types.NamespacedName) (*accountP
 	}
 	return accPoller, true
 }
+
+// getAccountConfig returns the account configuration.
+func (a *AccountManager) getAccountConfig(name *types.NamespacedName) *accountConfig {
+	a.mutex.RLock()
+	defer a.mutex.RUnlock()
+	config, ok := a.accountConfigMap[*name]
```

Comment:
use golang style lookup

Reply:
config will be used later in the function.